src/main/java/com/commercetools/sync/categories/CategorySync.java

```diff
@@ -607,61 +660,61 @@ static boolean requiresChangeParentUpdateAction(@Nonnull final Category category
-                                          sphereException,
-                                          () -> refetchAndUpdate(oldCategory, newCategory),
-                                          () -> {
-                                              if (!processedCategoryKeys.contains(categoryKey)) {
-                                                  handleError(format(UPDATE_FAILED, categoryKey, sphereException),
-                                                      sphereException, oldCategory, newCategory, updateActions);
-                                                  processedCategoryKeys.add(categoryKey);
-                                              }
-                                              return CompletableFuture.completedFuture(null);
-                                          });
-                                  } else {
-                                      if (!processedCategoryKeys.contains(categoryKey)) {
-                                          statistics.incrementUpdated();
-                                          processedCategoryKeys.add(categoryKey);
-                                      }
-                                      if (categoryKeysWithResolvedParents.contains(categoryKey)) {
-                                          statistics.removeChildCategoryKeyFromMissingParentsMap(categoryKey);
-                                      }
-                                      return CompletableFuture.completedFuture(null);
-                                  }
-                              });
+            .handle((updatedCategory, sphereException) -> sphereException)
+            .thenCompose(sphereException -> {
+                if (sphereException != null) {
+                    return executeSupplierIfConcurrentModificationException(
+                        sphereException,
+                        () -> refetchAndUpdate(oldCategory, newCategory),
+                        () -> {
+                            if (!processedCategoryKeys.contains(categoryKey)) {
+                                handleError(format(UPDATE_FAILED, categoryKey, sphereException),
+                                    sphereException, oldCategory, newCategory, updateActions);
+                                processedCategoryKeys.add(categoryKey);
+                            }
+                            return CompletableFuture.completedFuture(null);
+                        });
+                } else {
+                    if (!processedCategoryKeys.contains(categoryKey)) {
+                        statistics.incrementUpdated();
+                        processedCategoryKeys.add(categoryKey);
+                    }
+                    if (categoryKeysWithResolvedParents.contains(categoryKey)) {
+                        statistics.removeChildCategoryKeyFromMissingParentsMap(categoryKey);
+                    }
+                    return CompletableFuture.completedFuture(null);
+                }
+            });
```

Comment:
Have you changed this or is it formatting issue?

Reply:
this was a formatting issue